internal/functions/Invoke-Command2.ps1

```diff
@@ -92,7 +92,7 @@ function Invoke-Command2 {
                 Authentication = $Authentication
                 Name           = $sessionName
                 ErrorAction    = 'Stop'
-                UseSSL         = (Get-DbatoolsConfigValue -FullName 'PSRemoting.PsSession.UseSSL' -Fallback $false)
+                UseSSL         = $UseSSL
```

Comment:
The parameter should be removed if we are not going to need `-UseSSL` on the function anymore.

Reply:
Yeah that's the other approach, only issue would be if config has $true but you want to override to $false. Honestly can't think of why you would do that though.

Reply:
The parameter no longer does anything now with the config being set here.